newhelm/runners/simple_test_runner_cli.py

```diff
@@ -33,13 +33,21 @@
     default=False,
     help="Disable caching.",
 )
+@click.option(
+    "--no-progress-bar",
```

Comment:
I know this was a one off for a specific situation and looks good for that. But I was curious if with this additional and the existence of the `--no-caching` flag that we are heading towards a global settings context situation.

Click also has come built in context manager functionality as well:
https://click.palletsprojects.com/en/8.1.x/api/#context

Out of scope for this PR, but just mentioning it.

Reply:
Neat. I don't think we need it quite yet, but I'll definitely keep it in mind.